GroversAlgorithm/ReferenceImplementation.qs

```diff
@@ -13,6 +13,7 @@ namespace Quantum.Kata.GroversAlgorithm {
     open Microsoft.Quantum.Arrays;
     open Microsoft.Quantum.Intrinsic;
     open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math; // To use PI
```

Comment:
I wouldn't leave a comment for the import - we usually import libraries when they are needed without much justification.
```suggestion
    open Microsoft.Quantum.Math;
```

Reply:
Done